packages/allure-mocha/test/spec/api/runtime/legacy/parameters.test.ts

```diff
@@ -48,7 +48,7 @@ describe("legacy runtime parameters API", () => {
         parameters: [
           {
             name: "foo",
-            // eslint-disable-next-line
+            // eslint-disable-next-line @typescript-eslint/quotes
             value: '{"key":10}',
```

Comment:
Just FYI, you can use `String.raw`{"key":10}` to bypass the linter and formatter